pyxnat/core/derivatives/ashs.py

```diff
@@ -0,0 +1,47 @@
+XNAT_RESOURCE_NAME = 'ASHS'
+
+def stats(self, mode='corr_nogray'):
+    import pandas as pd
+
+    f = list(self.files('*icv.txt'))[0]
+    uri = f._uri
+    res = self._intf.get(uri).text.split(' ')
+
+    f = list(self.files('*_left_%s_volumes.txt'%mode))[0]
+    uri = f._uri
+    resl = self._intf.get(uri).text.split('\n')
+
+    f = list(self.files('*_right_%s_volumes.txt'%mode))[0]
+    uri = f._uri
+    resr = self._intf.get(uri).text.split('\n')
+
+    volumes = ['CA1', 'CA2', 'DG', 'CA3', 'misc', 'SUB',
```

Comment:
This list of subfield labels is not used, consider removing it.

Reply:
bb889094fa fixes this.